react-nginx/README.md

```diff
@@ -5,7 +5,7 @@
 Project structure:
```

Comment:
Missing the rename of the docker-compose.yaml for this sample

Reply:
indeed, I searched for docker-compose.yaml, not docker-compose.yml